python/datasources/cdc_vaccination_national.py

```diff
@@ -23,19 +24,37 @@
     'Race_eth_NHNHOPI': Race.NHPI_NH.value,
     'Race_eth_NHWhite': Race.WHITE_NH.value,
     'Race_eth_unknown': Race.UNKNOWN.value,
+    'US': Race.TOTAL.value,
 }
 
 CDC_AGE_GROUPS_TO_STANDARD = {
-    'Ages_<12yrs': '0-11',
-    'Ages_12-15_yrs': '12-15',
-    'Ages_16-17_yrs': '16-17',
+    'Ages_5-11_yrs': '5-11',
+    'Ages_12-17_yrs': '12-17',
     'Ages_18-24_yrs': '18-24',
     'Ages_25-39_yrs': '25-39',
     'Ages_40-49_yrs': '40-49',
     'Ages_50-64_yrs': '50-64',
     'Ages_65-74_yrs': '65-74',
     'Ages_75+_yrs': '75+',
     'Age_unknown': 'Unknown',
+    'US': std_col.TOTAL_VALUE,
+}
+
+
+# The CDC uses age ranges that we can not calculate with the given acs data,
+# and they don't publish these population numbers directly anywhere, so I am
+# taking the population percentages directly off of the chart here:
+# https://covid.cdc.gov/covid-data-tracker/#vaccination-demographic
```

Comment:
i haven't looked at front end yet, but is this notated for the user that we are using these numbers?

Reply:
On the methodology page: 

>> For the national numbers we use the population numbers provided by the CDC, we chose to do this because they include population estimates from Palau, Micronesia, and the U.S. Marshall Islands, which are difficult to find estimations for. Furthermore the CDC has estimations for age ranges that the ACS numbers do not readily provide, as they use a per year population estimate from the ACS that we do not use anywhere else and have not added to our system.

Reply:
Cool; should we hyperlink that "population numbers provided by the CDC" to https://covid.cdc.gov/covid-data-tracker/#vaccination-demographic ?